.github/workflows/functional_tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Run functional tests
```

Comment:
Now you need to make this check green by checking output of all commands and somehow checking if they failed or diktat found errors.